content/pages/download.md

```diff
@@ -71,6 +71,55 @@ versions:
         - slug: source
           name: 2.3.5 release
           file_url: https://github.com/mixxxdj/mixxx/archive/2.3.5.tar.gz
+  beta:
+    name: 2.4-beta
+    title: Beta Snapshots
+    text: |
+      A great way to contribute to Mixxx is testing the upcoming version before is has been released. Refer to the [Testing wiki page](https://github.com/mixxxdj/mixxx/wiki/Testing) for where to find the latest builds and instructions how to test pull requests before they are merged.
+
```

Comment:
I think we should emphasize that we prefer the beta snapshot being tested over the development snapshot.

Reply:
Yes, currently both texts are kind of duplicated. Not sure if it woks if we short the text for the alpha to a minimum. 
Ideas?

Reply:
How about we imply that testing the beta is usually preferred? something like this:

```
If you're bored of thoroughly testing the beta snapshots - or there may not be a beta running currently - you can also test the current development snapshot to get a glimpse at the even newer features. As for the beta snapshots, please refer to the [Testing wiki page](https://github.com/mixxxdj/mixxx/wiki/Testing) for where to find the latest builds and instructions how to test pull requests before they are merged.
```